backend/core/scripts/listings-importer.ts

```diff
@@ -70,24 +65,27 @@ async function uploadAmiCharts(units) {
   }
 }
 
-async function linkToUnitTypes(units) {
+interface T {
+  unitType?: any
```

Comment:
nit: I think we can swap in `string` for `any`?

Reply:
The generic object provided (either a `unit` or `unitsSummary`) will have a prop `unitType` that is initially a `string` (that's what it's set to on initial import) and is later set to a `UnitTypeCreate` or `UnitType` (from `backend-swagger`). So I could change this to `string | UnitTypeCreate | UnitType` but `any` seemed cleaner. But lmk if you disagree!

Reply:
I added a comment to clarify.